src/model/TransactionsExplorer.ts

```diff
@@ -113,6 +113,13 @@ export class TransactionsExplorer {
 			return false;
 		}
 		return parseInt(rawTransaction.vout[0].amount) !== 0;
```

Comment:
This line need to be removed or the addition (try/catch) will be useless.
It should then be fine